iOS error compilation version facebook_app_events: ^0.13.1

Hi Dennis, the error is now only in this line:

Xcode's output:
↳
    /Users/assert/.pub-cache/hosted/pub.dartlang.org/facebook_app_events-0.13.1/ios/Classes/SwiftFacebookAppEventsPlugin.swift:13:9: error: instance member 'initializeSDK' cannot be used on type 'ApplicationDelegate'; did you mean to use a value of this type instead?
            ApplicationDelegate.initializeSDK()
            ^~~~~~~~~~~~~~~~~~~
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    Command CompileSwift failed with a nonzero exit code
    note: Using new build system
    note: Building targets in parallel
    note: Planning build
    note: Analyzing workspace
    note: Constructing build description
    note: Build preparation complete


thanks for the effort!

--- a/ios/Classes/SwiftFacebookAppEventsPlugin.swift
+++ b/ios/Classes/SwiftFacebookAppEventsPlugin.swift
@@ -10,7 +10,7 @@ public class SwiftFacebookAppEventsPlugin: NSObject, FlutterPlugin {
         // Required for FB SDK 9.0, as it does not initialize the SDK automatically any more.
         // See: https://developers.facebook.com/blog/post/2021/01/19/introducing-facebook-platform-sdk-version-9/
         // "Removal of Auto Initialization of SDK" section
-        ApplicationDelegate.initializeSDK()
+        ApplicationDelegate.shared.initializeSDK()
 
         registrar.addMethodCallDelegate(instance, channel: channel)
     }

